src/content/docs/en/reference/programmatic-reference.mdx

```diff
@@ -157,6 +157,39 @@ await build({
 });
 ```
 
+### `BuildOptions`
+
+```ts
+export interface BuildOptions {
+	devOutput?: boolean;
+	teardownCompiler?: boolean;
+}
+```
+
+#### `devOutput`
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `false`
+<Since v="5.4.0" />
+</p>
+
+Output a development-based build similar to code transformed in `astro dev`. This can be useful to test build-only issues with additional debugging information included.
+
+#### `teardownCompiler`
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `true`
+<Since v="5.4.0" />
+</p>
+
+Teardown the compiler WASM instance after build. This can improve performance when building once, but may cause a performance hit if building multiple times in a row.
```

Comment:
```suggestion
Teardown the compiler WASM instance after build. This can improve performance when building once but may cause a performance hit if building multiple times in a row.
```